src/components/Explore/TokenRow.tsx

```diff
@@ -0,0 +1,275 @@
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+    display:none
+  `};
+`
+const ListNumberContainer = styled(Cell)`
+  flex-direction: column;
+  padding: 14px 0px;
+  gap: 10px;
+  color: ${({ theme }) => theme.text2};
+`
+const NameContainer = styled(Cell)`
+  justify-content: flex-start;
+  padding: 14px 0px;
+  gap: 8px;
+  min-width: 200px;
+`
+const PriceContainer = styled(Cell)`
+  justify-content: flex-end;
+  align-items: center;
+  padding: 12px 0px;
+  gap: 10px;
+`
+const PercentChangeContainer = styled(Cell)`
+  flex-direction: column;
+  align-items: flex-end;
+  padding: 14px 0px;
+  gap: 10px;
+  min-width: max-content;
+`
+
+const MarketCapContainer = styled(Cell)`
+  justify-content: flex-end;
+  padding: 12px 0px;
+  gap: 10px;
+  min-width: max-content;
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+  display: none;
+`};
+`
+const VolumeContainer = styled(Cell)`
+  justify-content: flex-end;
+  padding: 12px 0px;
+  gap: 10px;
+  ${({ theme }) => theme.mediaWidth.upToMedium`
+    display: none;
+  `};
+`
+const SparkLineContainer = styled(Cell)`
+  flex-direction: column;
+  padding: 16px 24px;
+  gap: 10px;
+  min-width: 120px;
+  ${({ theme }) => theme.mediaWidth.upToLarge`
+    display: none;
+  `};
+`
+const SparkLineImg = styled(Cell)`
+  max-width: 124px;
+  max-height: 28px;
+  flex-direction: column;
+  transform: scale(1.2);
+`
+
+const SwapContainer = styled(Cell)`
+  flex-direction: column;
+  padding: 16px 0px;
+  gap: 10px;
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+  display:none
+`};
+`
+
+const SwapButton = styled.button`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 8px;
+  gap: 6px;
+  font-weight: 600;
+
+  width: 54px;
+  height: 32px;
+
+  background: ${({ theme }) => theme.primary2};
+  border-radius: 12px;
+  border: none;
+  color: ${({ theme }) => theme.white};
+  &:hover {
+    background-color: ${({ theme }) => darken(0.05, theme.primary2)};
+  }
+`
+
+const TokenSymbol = styled.span`
+  color: ${({ theme }) => theme.text3};
+`
+const ArrowContainer = styled.div`
+  padding-left: 4px;
+  display: flex;
+  flex-direction: column;
+`
+
+const SortingCategory = styled.span`
+  color: ${({ theme }) => theme.primary1};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+
+const SortArrowContainer = styled(Cell)`
+  flex-direction: column;
+  padding-right: 2px;
+`
+
+function getHeaderCategory(category: string, sortCategory: string, sortDecreasing: boolean) {
+  if (sortCategory === category) {
+    return (
+      <SortingCategory>
+        <SortArrowContainer>{sortDecreasing ? <ArrowDown size={14} /> : <ArrowUp size={14} />}</SortArrowContainer>
+        {category}
+      </SortingCategory>
+    )
+  }
+  return category
+}
+
+export function headerRow() {
+  /* TODO: access which sort category used and timeframe used */
+  const possibleSortCategories = ['Market Cap', 'Price', '% Change']
```

Comment:
the categories should live at the top level as `SORT_CATEGORIES`

Reply:
i think you should make an enum for `Category` also and use that instead of strings